packages/web-components/fast-foundation/src/design-system/design-system.ts

```diff
@@ -300,6 +300,9 @@ class DefaultDesignSystem implements DesignSystem {
 
                     elementTypesByTag.set(elementName!, type);
                     elementTagsByType.set(type, elementName!);
+                    if (baseClass) {
```

Comment:
Yep. I think it's probably this simple and should address the problem. I think this will help with a couple of Fluent/FAST components as well that need an additional registration.